src/polygonContains.js

```diff
@@ -46,6 +46,8 @@ export default function(polygon, point) {
         var intersection = cartesianCross(normal, arc);
         cartesianNormalizeInPlace(intersection);
         var phiArc = (antimeridian ^ delta >= 0 ? -1 : 1) * asin(intersection[2]);
+        var pole = sin(phi);
+        if (pole == -1 || pole == 1) phi += pole * epsilon;
```

Comment:
Let’s use `===` here instead of `==`, and combine these two `var` declarations.

Reply:
The name `sinPhi` instead of `pole` might also be more descriptive.

Reply:
Actually, do we need to compute sin(*phi*) here? We’re basically undoing the asin(intersection[2]) we just computed, so why not use that?